### ISSUE TITLE
minor change in html report

### ISSUE THREAD

**Issue by tp0b** [geonetwork-delwp/metashare#28]
In Brief tab, Rename Currency Date to Publication Date and populate the value of publicationDate to there in the format DD Month YYYY e.g. 20 September 2021

Current html report
https://uat-metashare.maps.vic.gov.au/geonetwork/srv/api/records/88eef6b9-b847-57ee-9e4c-41d3b6bcf312/formatters/sdm-html?root=html&output=html

![image](https://user-images.githubusercontent.com/64397239/132795455-c786cf35-fa77-4406-baee-c23b93d501e5.png)

geonetwork url https://uat-metashare.maps.vic.gov.au/geonetwork/srv/eng/catalog.search#/metadata/88eef6b9-b847-57ee-9e4c-41d3b6bcf312

**Comment by tp0b** [comment]
9/11/2021 item 6, In Brief tab, Progress should get the value from collection progress. Possible values are: Complete, In Progress, Planned, Or Not Known. For this record, it should be displayed as Complete https://uat-metashare.maps.vic.gov.au/geonetwork/srv/api/records/a95c7ad8-f5d2-5a03-8ade-748b9fc1a03a/formatters/sdm-html?root=html&output=html-

item 13. Add a blank line between each field line.

15. Add UUID  row above ANZLICID in every report.

**Comment by tp0b** [comment]
@milleri we finalise this html report requirement. Can you please pass it to Tom? Thanks.

**Comment by thollands** [comment]
Hi @tp0b, going through some of these changes today and I have a couple of questions. CC @milleri 

> In Brief tab, Progress should be dataset collection status, but for this record it is displayed as Victoria
task 6 - Victoria is definitely wrong and needs to fixed, but this information is already in the field directly above called "Dataset status". Is this a duplication or is there some other value that needs to go in there?
![image](https://user-images.githubusercontent.com/18300553/144796157-d1da16c7-b4e5-4a07-aab8-27e9baf384f0.png)

> 7. In Brief tab, Access Constraint uses the value from Constraints/Use Limitation
task 7 - if that is not the correct field, which should it be?

> 12. Stored Data Format populate the format value
task 12 - the `distributionFormat` value is already being used under "Available Format(s) Types:". I cannot find another field which mentions data formats.

> 14. Confirm if content of Lineage section is a duplication of Data Source section
yes, they are duplicated, will remove.

I think that's it! Thanks in advance!

**Comment by tp0b** [comment]
hi @thollands thanks for your time working on this. 
Task 6: no it is not duplicated. It is collection status whereas the other is dataset status. As per my message dated on 9 November 2021, the possible value for this collection status is different from dataset status. This is the screenshot of MetaShare onpremise system 
![image](https://user-images.githubusercontent.com/64397239/144799437-fe578c98-82fd-4250-bf97-43fc7f7904c4.png)
Task 7 I need @emilskender and @spatialgeorge to confirm if they have rule to display creative common license similar like current SMD 
![image](https://user-images.githubusercontent.com/64397239/144799658-e554d368-f4bb-48b8-b292-d5b43e6bec4e.png)
Task 12 it seems to me that migration from on premise MetaShare to Geonetwork uses Available format Type rather than Stored Data Format, this is the screenshot . @spatialgeorge is it critical to be fixed before Tom can update the html report. 
![image](https://user-images.githubusercontent.com/64397239/144799772-2f9e5bef-c690-4e6a-9bcc-097200e48296.png)

**Comment by thollands** [comment]
@tp0b thanks for the clarification. 

task 6 - in the example record you provided this data doesn't exist. It the "progress" value is probably not being extracted through METAR.
![image](https://user-images.githubusercontent.com/18300553/144929067-29bffae2-5039-499f-9ebe-a58b0d5e0a37.png)

task 7 - ok I've found the CC attribution text in the metadata record. We can display this if you like.

task 12 - sounds right - all I can see on the record is "DIGITAL" so must just be the available format

**Comment by spatialgeorge** [comment]
@thollands Yes, can we apply the access constraints please as in the image supplied by Thu.

As for the Collection Progress, there are a bunch of attributes listed, most common is Complete. Definitely not Victoria

**Comment by thollands** [comment]
@spatialgeorge yes I understand that there are multiple values for the progress field, what I'm saying is that this data is not on the xml record, so I can't display it. The only values I can find which resemble these are the "status" value that is current displayed

**Comment by tp0b** [comment]
@thollands can you display Comments field in the html report too for example for VMADD_ADDRESS (ANZVI0803003126) Attributes TAB
https://metashare.maps.vic.gov.au/geonetwork/srv/api/records/8f3e3caa-e4bd-5e43-afc4-59b71c1f27f2/formatters/sdm-html?root=html&output=html

this is an example of SDM html report for this dataset
![image](https://user-images.githubusercontent.com/64397239/148474287-e48061b0-a7b9-450e-9fc3-2338fea976d8.png)
